### PR TITLE
Add bower.json

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webcomponents-platform",
+  "version": "1.0.0-rc.1",
+  "description": "Basic platform polyfills needed by webcomponents.js",
+  "main": "webcomponents-platform.js",
+  "authors": [
+    "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)"
```
Comment (notwaldorf) [inline bower.json:7]: Just use "The Polymer Authors", please

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "keywords": [
+    "web-components",
+    "webcomponents",
+    "polyfill",
+    "shim"
+  ],
+  "homepage": "https://webcomponents.org/polyfills",
+  "ignore": [
+    "**/.*",
```
Comment (notwaldorf) [inline bower.json:22]: Why is this generic wildcard needed?

**Review by notwaldorf — CHANGES_REQUESTED** [review]

File: bower.json (hunk 2 above)
Comment (FluorescentHallucinogen) [reply bower.json:22]: It's needed for excluding files like `.travis.yml`, `.gitignore`, etc. from Bower package.

**Comment by notwaldorf** [comment]
Closing, this is an old PR with unaddressed requested changes.

**Pull request closed without merging**